great_expectations/data_context/data_context/cloud_data_context.py

```diff
@@ -74,27 +79,43 @@ def __init__(
             project_config (DataContextConfig): config for CloudDataContext
             runtime_environment (dict):  a dictionary of config variables that override both those set in
                 config_variables.yml and the environment
-            ge_cloud_config (GeCloudConfig): GeCloudConfig corresponding to current CloudDataContext
+            cloud_config (GXCloudConfig): GXCloudConfig corresponding to current CloudDataContext
```

Comment:
Love that we are using all caps for abbreviations here.

Reply:
still missing some instances but I'll clean em up as we go 😄